msgraph-mail/php/composer.json

```diff
@@ -0,0 +1,22 @@
+{
```

Comment:
can you duplicate the project files (not the generated stuff) under /template/php please? should anyone want to provide a sample with a different API.